devicetypes/fibargroup/fibaro-door-window-sensor-zw5.src/fibaro-door-window-sensor-zw5.groovy

```diff
@@ -211,7 +242,8 @@ def configure() {
     cmds += zwave.versionV1.versionGet()
```

Comment:
It's not really related to this PR, but can we take out manufacturerSpecificGet and versionGet since that info is in the Raw Description now